assets/_scss/components/_forms.scss

```diff
@@ -129,3 +148,14 @@ input.usa-input-medium {
   font-style: italic;
   font-weight: normal;
 }
+
+.usa-input-buttons_inline {
+  button[type="submit"],
+  input[type="submit"] {
+    display: inline;
+  }
+  
+  .usa-button-cancel {
```

Comment:
Adding margin to the right of the button might be a fix that could apply to everything.